### ISSUE TITLE
Issue with whitelist not working. 

### ISSUE THREAD

**Issue by andrewclacko** [prowler-cloud/prowler#598]
whitelist not working as expected and still coming through as FAIL and not WARN.
any ideas anyone or an example of a working one?

command ran :  prowler -c check121 -w prowler_whitelist.txt
format as below - prowler_whitelist.txt (tried numerous combinations as you can see)

check121:"User email has never used access key 1 in over 90 days"
check121:User email has never used access key 1 in over 90 days
check121:1.21:arn:aws:iam::account:user/email
check121:1.21:"arn:aws:iam::account:user/email"

1.21:arn:aws:iam::464177228504:user/email
1.21:User email has never used access key 1 in over 90 days
1.21:"User email has never used access key 1 in over 90 days"

**Comment by toniblyx** [comment]
Hi @andrewclacko whitlist works for those checks that get resource ID (buckets, instances, security-groups, etc). @urjitbhatia could you please clarify this question? We may need to add more context or examples to the sample `prowler_whitelist.txt`.

**Comment by urjitbhatia** [comment]
Yeah @andrewclacko it should work for things that are named resources.
For example:
`check121:arn:aws:iam::account:user/email` should work.
I wonder if there is a bug parsing multiple `:` in the lines - let me check that.

**Comment by urjitbhatia** [comment]
Can you add a `printenv` in the check here: https://github.com/toniblyx/prowler/blob/master/checks/check121#L20
and check the value of `IGNORES=` ?
That should contain the whitelist elements for that check.

I checked that multiple `:` in the whitelist works fine

**Comment by urjitbhatia** [comment]
ah, I think I know what happened.
That check outputs lines like:
```
FAIL! User foo has never used access key 1
FAIL! User bar has never used access key 1
```
The whitelist checks run on the output of the check. So `arn:aws:iam::account:user/foo` is not matched.
Try:
```
check121:user1
check121:user2
```

**Comment by toniblyx** [comment]
This is fixed in #616
